src/App.js

```diff
@@ -87,39 +90,46 @@ const App = () => {
 	}
 
 	//Gameplay
-	const levelMarks = [1, 2, 4, 8, 12, 18, 24, 36, 48, 60, 75, 100, 125, 150, 175, 200, 225, 260, 300, 340, 375, 425, 500]
```

Comment:
This seemed pretty arbitrary to me, and didn't make things super easy when trying to find the current level based on xp